Tests/PrettyPrinterTests/ProtocolDeclTests.swift

```diff
@@ -0,0 +1,133 @@
+public class ProtocolDeclTests: PrettyPrintTestCase {
+  public func testBasicProtocolDeclarations() {
+    let input =
+      """
+      protocol MyProtocol {
+        let A: Int
```

Comment:
Protocols can't have `let` decls; they must be declared `var` and have a get/set block indicating whether the getter and/or setter must be implemented. For example:

```swift
protocol Foo {
  var foo: Int { get }
  var foo: Int { get set }
}
```

(You can't have `set` without `get`, AFAIK.)

We play a little loose with semantically valid Swift here, but this is a case where the syntax is unique enough that we probably want to verify it more accurately. If either of those above wrapped, I imagine we'd want this:

```swift
protocol Foo {
  var foo: Int {
    get
  }
  var foo: Int {
    get
    set
  }
}
```

But maybe `get set` could go together on the same line in that last one too. Since they don't have bodies, I don't feel particularly strong about that part.

Reply:
I did a quick check, and it looks like sets must come with gets. Also, I added the wrapping from your second example in the unlikely event those lines need to wrap.